syntaxes/injection.json

```diff
@@ -0,0 +1,11 @@
+{
+    "scopeName": "memo.wikilink.injection",
+    "injectionSelector": "L:meta.paragraph.markdown",
+    "patterns": [
+      {
+        "contentName": "string.other.link.title.markdown.memo",
```

Comment:
A little explanation may be in order here:

- `string.other.link.title.markdown` is the current theme color for link titles in markdown files, the one `[HERE](not here)`. If you prefer `[not this](but THiS)` it's easy to change, but since wikilinks are more like titles than like urls I believe this is the right call. It was the color adopted by foam, also.
- `string.other.link.title.markdown.memo` inherits `string.other.link.title.markdown` but allows for customization using `string.other.link.title.markdown.memo`. This means we get a sensible default that changes with current theme but the user can customize it, even in a theme dependent way, without the need to expose any new setting from memo itself.